CONTRIBUTING.md

```diff
@@ -1,5 +1,3 @@
 # Contributing
 
-Pants Developer Center: http://www.pantsbuild.org/dev.html (hacking on Pants itself)
-
-Pants Contributors Guide: http://www.pantsbuild.org/howto_contribute.html (interact with pantsbuild community)
+Please see [pants.readme.io/docs/contribution-flow](https://pants.readme.io/docs/contribution-flow).
```

Comment:
I'm not certain about this URL slug. Is this what we want to commit to? Possible alternatives: `contribution-overview` or `contributors-overview`.

Reply:
The latter is nice. But probably fine either way.